docs/UserGuide.adoc

```diff
@@ -12,7 +12,7 @@ ifdef::env-github[]
 :tip-caption: :bulb:
 :note-caption: :information_source:
 endif::[]
-:repoURL: https://github.com/se-edu/addressbook-level3
+:repoURL: https://github.com/AY1920S1-CS2103-T16-3/main
```

Comment:
The link in `src/main/java/seedu/weme/ui/HelpWindow.java` should also be updated.